docker-compose.yaml

```diff
@@ -0,0 +1,32 @@
+services:
+  cinema:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8001:8000"
+    volumes:
+      - ./:/backend

```

Comment:
Mounting the entire project directory as a volume can lead to potential issues with file permissions and unnecessary files being included in the container. Consider specifying only the necessary directories or files.